advocate_europe/assets/scss/components/_call_to_action.scss

```diff
@@ -0,0 +1,17 @@
+.call-to-action-tile.idealist-item {
+    background-color: $brand-primary;
+    padding: rem(20px 12px);
+    align-items: center;
+
+    .call-to-action-link {
+        color: $gray-dark;
+        text-decoration: underline;
+    }
+
+    .call-to-action-btn {
+        margin-left: rem(20px);
```

Comment:
There must be a better way! But without that the button was always as wide as the tile (minus the tile's padding).